node/pkg/governor/governor.go

```diff
@@ -170,6 +238,7 @@ func (gov *ChainGovernor) initConfig() error {
 
 	gov.dayLengthInMinutes = 24 * 60
 	configTokens := tokenList()
+	flowCancelTokens := FlowCancelTokenList()
```

Comment:
Something to think about. We have different lists of monitored tokens for devnet and testnet. Should we have something similar for flow cancelled tokens?

Reply:
Yeah that seems sensible. I'll look into this today